.github/workflows/mkdocs.yml

```diff
@@ -1,20 +0,0 @@
-name: publish-docs
-on:
-  push:
-    branches:
-      - main
```

Comment:
☝️

Reply:
Yup. I redid and force pushed the second commit where I did the consolidation.